rholang/src/main/scala/coop/rchain/rholang/interpreter/util/AddressTools.scala

```diff
@@ -0,0 +1,61 @@
+package coop.rchain.rholang.interpreter.util
+
+import java.nio.charset.StandardCharsets
+
+import coop.rchain.crypto.hash.Blake2b256
+import coop.rchain.rholang.interpreter.util.codec.Base58
+
+final case class Address(val prefix : Array[Byte], val heyHash : Array[Byte])
```

Comment:
we need to fix this typo in `heyHash` :)

Reply:
nice catch! :))